index.bs

```diff
@@ -220,8 +250,23 @@ spec: webrtc-pc; urlPrefix: https://www.w3.org/TR/webrtc/#
       </p>
 
       <p>
-        A <dfn>valid VideoConfiguration contentType</dfn> MUST include a media
-        type, subtype and full codecs description.
+        To check if a {{VideoConfiguration}} <var>configuration</var> is a
+        <dfn>valid video configuration</dfn>, the following steps MUST be run:
+        <ol>
+          <li>
+            If <var>configuration</var>'s {{VideoConfiguration/contentType}} is
+            not a <a>valid video MIME type</a>, return <code>false</code> and
+            abort these steps.
+          </li>
+          <li>
+            If <var>configuration</var>'{{VideoConfiguration/framerate}} is not
```

Comment:
Is the intention that a missing frameramte is treated as invalid? That's how I read this, but that doesn't match the definition of "valid video MIME type".